src/manager.rs

```diff
@@ -438,34 +339,20 @@ where
         ctx.notify(Notification(msg.into()));
     }
 
-    fn set_timeout(
-        &self,
-        ctx: &SmContext<K>,
-        id: StateId<K>,
-        duration: Duration,
-    ) -> Result<(), SendError<Notification<K::Message>>> {
+    fn set_timeout(&self, ctx: &SmContext<K>, id: StateId<K>, duration: Duration) {
         ctx.notification_queue
-            .send(Notification(TimeoutInput::set_timeout(id, duration).into()))
+            .priority_send(Notification(TimeoutInput::set_timeout(id, duration).into()))
     }
 
-    fn set_timeout_millis(
-        &self,
-        ctx: &SmContext<K>,
-        id: StateId<K>,
-        millis: u64,
-    ) -> Result<(), SendError<Notification<K::Message>>> {
-        ctx.notification_queue.send(Notification(
+    fn set_timeout_millis(&self, ctx: &SmContext<K>, id: StateId<K>, millis: u64) {
+        ctx.notification_queue.priority_send(Notification(
```

Comment:
This is a nit, but could call `self.set_timeout(ctx, id, Duration::from_mills(millis));` Then delete `UnaryRequest::set_tmeout_mills` and `Operation::from_millis`